plugins/modules/scc_product.py

```diff
@@ -58,12 +56,12 @@
 
 
 from ansible_collections.theforeman.foreman.plugins.module_utils.foreman_helper import KatelloAnsibleModule
-
+from distutils.version import LooseVersion
```

Comment:
`plugins/modules/scc_product.py:61:1: E302 expected 2 blank lines, found 1`

the linter is picky, you need another empty line after the import.

Reply:
Fixed!